docs/scripts/buildApi.ts

```diff
@@ -372,7 +372,7 @@ async function buildDocs(options: {
   // no Object.assign to visually check for collisions
   reactAPI.forwardsRefTo = testInfo.forwardsRefTo;
 
-  // if (reactAPI.name !== 'TableCell') {
+  // if (reactAPI.name !== 'Timeline') {
   //   return;
   // }
```

Comment:
While we're at it we should remove this block. You can use `yarn docs:api:build --grep "/Timeline/"`

Reply:
Oh nice, I forgot this existed.

Reply:
I'll try to support `--grep` on most CLIs we have where it makes sense i.e. where you're often debugging an isolated issue. `proptypes` has this as well and `mocha` already has it built-in.